code/style.css

```diff
@@ -40,19 +73,157 @@ p {
 .room-deck {
   width: 250px;
   height: 400px;
-  background: midnightblue;
-
+  background: #900d0d;
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
 }
 
 button {
-  padding: 8px 16px;
-  background: crimson;
-  border: none;
+  padding: 16px 25px;
+  background: #900d0d;
+  border: 2px solid #900d0d;
   font-size: 20px;
-  border-radius: 8px;
+  border-radius: 20px;
   display: block;
   margin: 0 auto;
+  color: whitesmoke;
+}
+
+button:after {
+  border: none;
+}
+
+#revealButton {
+  text-align: center;
+}
+
+#restartButton {
+  display: none;
+}
+
+article, #revealButton, #restartButton {
+  cursor: pointer;
+}
+
+/*** Animation styling ***/
+
+.loader-wrapper {
+  display: none;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding-top: 50px;
+}
+
+.loader {
+  border: 16px solid #f3f3f3;
+  border-radius: 50%;
+  border-top: 16px solid lightslategrey;
+  width: 150px;
+  height: 150px;
+  -webkit-animation: spin 2s linear infinite;
+  animation: spin 2s linear infinite;
+}
+
+@-webkit-keyframes spin {
+  0% {
+    -webkit-transform: rotate(0deg);
+  }
+  100% {
+    -webkit-transform: rotate(360deg);
+  }
 }
+
+@keyframes spin {
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+.start-display-cards {
+  background-color: whitesmoke;
+  padding: 15px;
+  border-radius: 10px;
+}
+
+.generating-text {
+  font-size: 25px;
+  padding: 25px;
+}
+
+/* Card styling */
```

Comment:
Really nice work with indentation and grouping the style for your different elements in your CSS-file 👏 . 
It's good to be consistent when doing this and having it follow the structure in the HTML as much as possible. 
I usually also start with declaring styles for the overall page first (like html, body, typography, and so on) and then go on to more specific styling for different sections, (articles, cards, buttons, footer, etc) just as you've done 👍  

Maybe the .cards belongs in /* Card styling */ as well? And the styling for p and h1  belongs together with the rest of the font styling in your section /* Font styling */ ?  :)